include/download_client.h

```diff
@@ -33,7 +33,9 @@ extern "C" {
 enum download_client_evt_id {
 	/** Event contains a fragment. */
 	DOWNLOAD_CLIENT_EVT_FRAGMENT,
-	/** Download complete. */
+	/** Download complete.
+	 *  The socket is closed automatically.
```

Comment:
Is it? I haven't seen any code added that would do that.

Reply:
You are right, I got confused; the socket is never closed automatically, it is left to the application.
Fixed.